notes-service/src/main/java/org/exoplatform/wiki/jpa/search/WikiElasticSearchServiceConnector.java

```diff
@@ -70,6 +70,15 @@ public class WikiElasticSearchServiceConnector extends ElasticSearchServiceConne
 
   private String                     searchQueryFilePath;
 
+  public static final String           SEARCH_QUERY_TERM            = ",\"must\":{" +
+          "  \"query_string\":{" +
+          "    \"fields\": [\"name\",\"title\",\"content\",\"comment\",\"attachment.content\"]," +
+          "    \"default_operator\": \"AND\"," +
+          "    \"query\": \"@term@\"" +
+          "  }" +
+          "}";
```

Comment:
You can use `triple double quote sequence` to simplify reading and maintenance:

```suggestion
  public static final String           SEARCH_QUERY_TERM            = """
         ,"must":{
            "query_string":{
              "fields": ["name","title","content","comment","attachment.content"],
              "default_operator": "AND",
              "query": "@term@"
            }
          }
  """;
```

Reply:
done